templates/fedora.tpl.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
       Template for {{ lookup('osinfo', oslabels[0]).name }} VM or newer.
       A PVC with the Fedora disk image must be available.
       Recommended disk image:
-      https://download.fedoraproject.org/pub/fedora/linux/releases/32/Cloud/x86_64/images/Fedora-Cloud-Base-32-1.6.x86_64.qcow2
+      https://download.fedoraproject.org/pub/fedora/linux/releases/34/Cloud/x86_64/images/Fedora-Cloud-Base-34-1.2.x86_64.qcow2
```

Comment:
Currently the fedora templates supports versions 33, 34, 35. I would vote for using version 33 or 35 image.

Reply:
Changed to 35